rust/basic_dao/README.md

```diff
@@ -0,0 +1,222 @@
+# Basic DAO
+
+This sample project demonstrates a basic DAO (Decentralized Autonomous Organization) that can be deployed to the
+[Internet Computer](https://github.com/dfinity/ic) 
+
```

Comment:
I'd explain briefly what a DAO is, here or in the next section. Or point to wikipedia if there is a good description.